data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/source/opensearch/worker/PitWorkerTest.java

```diff
@@ -381,7 +419,9 @@ void run_with_getNextPartition_with_valid_existing_point_in_time_does_not_create
         verify(sourceCoordinator, times(0)).saveProgressStateForPartition(eq(partitionKey), eq(openSearchIndexProgressState));
         verify(sourceCoordinator, times(0)).updatePartitionForAcknowledgmentWait(anyString(), any(Duration.class));
 
+        verify(bytesReceivedSummary, times(3)).record(0L);
```

Comment:
This doesn't verify the behavior sufficiently. I think we should mock `ObjectMapper` to return a `byte[]` of sizes that we can check against.

e.g.

```
int expectedDataSize1 = 10;
int expectedDataSize2 = 20;
...
when(objectMapper.writeValueAsBytes(testData1)).thenReturn(expectedDataSize1);
...

verify(bytesReceivedSummary).record(expectedDataSize1);
verify(bytesReceivedSummary).record(expectedDataSize2);
...
```